intersect_registry_service/app/core/environment.py

```diff
@@ -78,6 +78,45 @@ class Settings(BaseSettings):
     If 'rudimentary' : use in-memory dictionary as the auth lookup
     """
 
+    AUTHORIZE_URL: str = ''
+    """
+    Authentication URL From the OIDC provider.
+    """
```

Comment:
Do we need to add an example of AUTHORIZE_URL to the `.env.example` file for the docker-compose yaml setup?
I was struggling a bit to set this up.

Reply:
You will also need to go into the keycloak interface and manually import the configuration from a file (the json file provided). The auto import command is really finicky. I also changed the name of the Realm to remove the the spaces. I can show you in a call if that's easier.

Reply:
Sure